core/src/plan/evaluable.rs

```diff
@@ -217,7 +213,6 @@ mod tests {
         test!("Bar.rate", true);
         test!("Foo.rate", false);
         test!("Rand.id", false);
-        test!("a.b.c", false);
```

Comment:
Looks like we have to fix unwrap at tester::expr to keep this unit test.

Reply:
It seems right to disappear as a patch in the current PR.